docs/r2_dataset.md

```diff
@@ -0,0 +1,77 @@
+## System Requirements
+> You will need to make sure you have adequate bandwidth. 17.51 TB of data (as of 2/6/2025) will be transferred through your machine but NOT be stored locally. The higher the internet speed and more workers/cores you can throw into the process, the faster the dataset will be finished. Consider that this process will take 12 or more hours, so use screen/tmux accordingly. Re-run the transfer command for the process to verify the files and pick up where it left off.
+
+Recommend workhorse:
+Network: 1gbps+
+Local Storage: 100gb
+RAM: 4 GB+ (this process is not memory intensive)
+Cores: 8+
+Estimated Download Time: 12–18 hours
+
+Instructions
+r2 bucket dataset population
+
+```bash
```

Comment:
I think every code should contain its own code block and text should be outside?